react/features/video-menu/components/native/RemoteVideoMenu.tsx

```diff
@@ -258,6 +264,7 @@ function _mapStateToProps(state: IReduxState, ownProps: any) {
         _disablePrivateChat: Boolean(disablePrivateChat) || _iAmVisitor,
         _isBreakoutRoom,
         _isParticipantAvailable: Boolean(isParticipantAvailable),
+        _isParticipantSilent: Boolean(isParticipantAvailable?.isSilent),
         _moderator: moderator,
```

Comment:
Can you rename, isParticipantAvailable to participant so it is obvious what it is :) Thanks.